src/pages/chainlink-nodes/configuring-nodes.mdx

```diff
@@ -0,0 +1,127 @@
+   ```
+
+   You will likely see some invalid config errors. For example:
+
+   ```shell
+   Invalid configuration: invalid configuration: EVM.3.Nodes: missing: must have at least one node
+   ```
+
+1. Edit the `config.toml` file and manually correct any errors. See the [Node Config](/chainlink-nodes/v1/node-config) and [Node Secrets](/chainlink-nodes/v1/secrets-config) pages to learn which config settings are valid. For the error in this example, an EVM chain was configured with no nodes. Removing this from the config made the config valid:
+
+   ```
+   [[EVM]]
+   ChainID = '421613'
+   Enabled = false
+   Nodes = []
+   ```
+
+1. Run the `config validate` command again and make additional changes until you have a valid config message:
+
+   ```shell
+   Valid configuration.
+   ```
+
+### Restart your Chainlink node using the TOML config
+
+With your valid config and secrets files, you can migrate your Chainlink node to use the new config.
+
+1. Stop your existing Chainlink node:
+
+   ```shell
+   docker stop chainlink
+   ```
+
+1. Make a Postgres [database snapshot](https://www.postgresql.org/docs/current/backup-file.html) so you can restore your previous Chainlink node if necessary.
+
+1. Start a new Chainlink node Docker container named using the new `config.toml` and `secrets.toml` files. This example uses `chainlink-toml` as the container name:
+
+   ```shell
+   docker run --platform linux/x86_64/v8 --name chainlink-toml  -v ~/.chainlink-toml:/chainlink -it -p 6688:6688 --add-host=host.docker.internal:host-gateway smartcontract/chainlink:1.13.0 local n -config /chainlink/config.toml -secrets /chainlink/secrets.toml
+   ```
+
+Test your node to verify that it works as intended. If you are using a VPS, open an [SSH tunnel](https://www.howtogeek.com/168145/how-to-use-ssh-tunneling/) using `ssh -i $KEY $USER@$REMOTE-IP -L 6688:localhost:6688 -N`. Connect to the Operator UI in your browser at `localhost:6688`.
+
+## Using multiple config files
+
+You can use multiple config and secrets files. The config settings from each file are applied in the order that you specify when you run your node. Duplicated fields override values specified in earlier config files. This allows you to create a common config that applies to many nodes with specific configs for nodes that need unique configuration settings. Specifying multiple secrets files is invalid.
+
+To specify multiple config files, add additional `-config` flags to the `docker run` command:
+
+```shell
+docker run --platform linux/x86_64/v8 --name chainlink-toml  -v ~/.chainlink-toml:/chainlink -it -p 6688:6688 --add-host=host.docker.internal:host-gateway smartcontract/chainlink:1.13.0 local n -config /chainlink/config.toml -config /chainlink/config2.toml -config /chainlink/config3.toml -secrets /chainlink/secrets.toml
```

Comment:
placement of `-config, -secret` as above

Reply:
Done